### PR TITLE
Bandaiding <table> overflow

### PR DESCRIPTION
**Pull request by ashleemboyer**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ambfix-table-overflow-hashicorp.vercel.app/vault/docs/internals/telemetry#token-identity-and-lease-metrics) 🔎
- [Asana task](https://app.asana.com/0/1202114367927919/1202439831418490/f) 🎟️

## 🗒️ What

- Temporarily fixes horizontal overflow of `<table>` elements with horizontal scrolling

## 🤷 Why

- On narrower viewports, a `<table>` wider than the viewport messes with the width of other content

## 🛠️ How

- CSS 🪄✨ (we mainly needed `display: block` and `overflow-x: auto`)

## 🧪 Testing

- [ ] Go to the preview link
- [ ] Make sure the tables horizontally scroll
- [ ] Make sure the other page content is appropriately sized

### CODE REVIEW

**Comment by ashleemboyer** [comment]
Well, 1280px doesn't work correctly 😆

CleanShot 2022-06-14 at 12 46 21@2x

File: src/components/dev-dot-content/dev-dot-content.module.css [hunk 1]
```diff
@@ -91,14 +91,16 @@
...
     border-spacing: 0;
     border: 1px solid #ddd;
-    border-right: 0;
-    background-color: transparent;
-    width: 100%;
-    max-width: 100%;
+    display: block;
     margin-bottom: 20px;
+    max-width: 100%;
+    overflow-x: auto; /* TODO: find alternative solution */
```
Comment (BRKalow) [inline src/components/dev-dot-content/dev-dot-content.module.css:102]: ⛏️ nit: I'm not sure what a _better_ solution would be 🤔

**Review by BRKalow — APPROVED** [review]

File: src/components/dev-dot-content/dev-dot-content.module.css (hunk 1 above)
Comment (ashleemboyer) [reply src/components/dev-dot-content/dev-dot-content.module.css:102]: Mainly something that doesn't involve horizontal scrolling. It makes content completely inaccessible to keyboard-only users.

Comment (BRKalow) [reply src/components/dev-dot-content/dev-dot-content.module.css:102]: Gotcha, can we specify that in the comment? That would be a helpful breadcrumb!

**Review by zchsh — APPROVED** [review]
✅ 🙌  Approving cause I think this is _much_ better, and a great band-aid, even if we don't remove `border-right`.

Had a nitpick, not really blocking but maybe close, re: CSS ordering to make `border-right: 0` effective.

**Comment by braydenlove** [comment]
Great solution! One thing I noticed was there seems to be a double border on the right side depending on where you are in the horizontal scroll. Here are a few screenshots. Not the end of the world but if we could remove that double border that would be great!

Double Border:
screenshot showing the double border

Single Border:
Different horizontal scroll position that doesn't show the double border

File: src/components/dev-dot-content/dev-dot-content.module.css [hunk 2]
```diff
@@ -91,14 +91,16 @@
   }
 
   & table {
+    background-color: transparent;
     border-collapse: collapse;
+    border-right: 0;
```
[the remark this answers is not in the document]
Comment (braydenlove) [reply src/components/dev-dot-content/dev-dot-content.module.css:96]: This also would solve for my double border comment below. I trust you all to make the right call here (on the eve of release).

**Pull request merged**